pkg/bundle/oidc.go

```diff
@@ -116,6 +116,10 @@ func formatRedirectUris(settings *api.OIDCSettings, ctx map[string]interface{})
 }
 
 func confirmOidc(confirm *bool) (bool, error) {
+	if confirm != nil && *confirm {
```

Comment:
It didn't really make sense to keep it at the end of function since it was never really used.